lib/delays.js

```diff
@@ -251,26 +251,44 @@ export function waitForAdditionalElemChildren(
  * @desc Waits for the number children of the supplied parentElement
  * to become greater than the supplied `currentChildCount`
  * @param {Element} parentElement - The parent
- * @param {WaitForOptions} [options]
+ * @param {GuardedWaitForOptions} [options]
  * @return {Promise<{predicate: boolean, maxExceeded: boolean}>}
  */
 export function waitForAdditionalElemChildrenMO(parentElement, options) {
-  const opts = Object.assign({ max: 15000 }, options);
-  let max = -1;
+  const { guard, max = 15000, pollRate = 1000 } = options || {};
```

Comment:
does it make sense to pull these values out and makes constants? eg. MAX_WAIT, POLL_RATE, etc..

Reply:
personally I think it is simplest to let these values be defined locally since the idiom with destructing is these values are per function